packages/analytics/lib/modular/index.js

```diff
@@ -64,6 +64,16 @@ export function initializeAnalytics(app, options) {
   return getApp(app.name).analytics();
 }
 
+/**
+ * Retrieves a unique Google Analytics identifier for the web client.
+ *
+ * @param {FirebaseAnalytics} analytics - Instance of analytics (web - only)
+ * @returns {string}
+ */
+export function getGoogleAnalyticsClientId(analytics) {
+  throw new Error('getGoogleAnalyticsClientId is web-only and not yet supported.');
```

Comment:
for web we have an implementation already (I'm aware because I was in there semi-recently - it works...)
https://github.com/invertase/react-native-firebase/blob/8f59579e587f8dda4fadef6360f27076007e636b/packages/analytics/lib/web/api.js#L165

if you haven't done it, running `yarn tests:macos:pod:install && yarn tests:macos:build && yarn tests:macos:test-cover` runs the "web" version, and if you put an `it.only` or `describe.only` on any part of the e2e suite it'll run just those so it's fast to test+iterate

shame not to expose it since that might be half the work that went into the "other" platform analytics!

Reply:
What I meant here was for `Platform.other`, it should actually return the result of the existing `_getCid()` method, because it does exist and does work. The error message `is web-only and not yet supported` would then be for the other branch of the conditional (where `Platform` was *not* `Platform.other`) and should be more like `is only supported on "other" platforms in react-native-firebase", because we do support it for "other" (macos, or anything else that uses our "other" platform support where we wrap the web SDK)